tests/accelerators/test_tpu_backend.py

```diff
@@ -50,7 +50,7 @@ def test_resume_training_on_cpu(tmpdir):
     """Checks if training can be resumed from a saved checkpoint on CPU."""
     # Train a model on TPU
     model = BoringModel()
-    trainer = Trainer(checkpoint_callback=True, max_epochs=1, tpu_cores=8)
+    trainer = Trainer(enable_checkpointing=True, max_epochs=1, tpu_cores=8)
```

Comment:
why's this needed if it's the default value? 
```suggestion
    trainer = Trainer(max_epochs=1, tpu_cores=8)
```

Reply:
I just did the replacements all at once.. you don't expect me to go, search and replace in each file .. haha :sweat_smile: 
thanks for pointing out btw.. :)